packages/mobile/src/components/Chat/Conversation-tile.tsx

```diff
@@ -33,6 +33,8 @@ type ConversationTileProps = {
   fileId?: string;
   payloadKey?: string;
   previewThumbnail?: EmbeddedThumb;
+  conversationCreated: number;
+  conversationUpdated: number;
```

Comment:
I could use that but  it doesn't make sense to refactor this again , since we already going through the props way of passing data here. Since any mutations, would already update this tile, I say let's not add a new hook in the tile and keep it as is :)

Reply:
Whatever you prefer, but please add it to a TODO list somewhere at least, as it can make a real performance difference and keep the code cleaner by doing so